Make sure that TGeo is using the old Root units

ihrivnac opened the pull request:
The default TGeo units (cm, s, GeV) has been changed in Root v6.24
to Geant4 defaults (mm, ns, MeV); while the VMC is based on the old system

olifre commented:
Just a small addition, since we also hit this issue: In ROOT 6.25.01, the default will be changed back, and the interface to switch the unit system has been different in earlier releases. 
See:
https://github.com/root-project/root/issues/9383
for details.

ihrivnac commented:
Hi @olifre,

thank you for pointing at the issue. I updated now this PR and added the Root version so that we do not block compilation with older ROOT versions. I checked v6-25-01 and the default is still TGeoManager::kG4Units; so I did not put a un upper version limit.

olifre commented:
Hi @ihrivnac ,

many thanks! We now also have such code in our framework, but I certainly appreciate `geant4_vmc` is also ensuring the consistent behaviour of units across ROOT versions. :+1:
